### PR TITLE
chore(releasing): Prepare 0.19.0 release

### PR DESCRIPTION
**Pull request by jszwedko**
Signed-off-by: Jesse Szwedko <jesse@szwedko.me>

### CODE REVIEW

File: website/cue/reference/releases/0.19.0.cue [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+
+				assert(.thing == "foo bar")
+				```
+				"""
+			pr_numbers: [10149]
+		},
+		{
+			type: "enhancement"
+			scopes: ["influxdb_logs sink"]
+			description: """
```
Comment (spencergilbert) [inline website/cue/reference/releases/0.19.0.cue:85]: Do we want to shoutout the contributor here? I've always thought that was nice.

Comment (jszwedko) [reply website/cue/reference/releases/0.19.0.cue:86]: 👍 Good idea. I pushed 8f41411 which adds another cue field for this. We'll need to update the website to render it though. I'll leave that as a follow-up.

**Comment by jszwedko** [comment]
Netlify doesn't seem to want to rebuild. I'll close and re-open this to see if that triggers it.

File: website/cue/reference/releases/0.19.0.cue [hunk 2]
```diff
@@ -124,7 +125,8 @@ releases: "0.19.0": {
 				ns.vector,metric_type=foo,host=example.com message="hello world",size=10 {timestamp}
 				```
 				"""
-			pr_numbers: [10082]
+			pr_numbers: [10082, 10217],
+			contributors: ["juvenn"],
```
Comment (spencergilbert) [inline website/cue/reference/releases/0.19.0.cue]: ❤️

**Comment by jszwedko** [comment]
Netlify refuses to rebuild 😢 I'll close this PR and open a new one.

**Pull request closed without merging**